nuclio/export.py

```diff
@@ -40,17 +40,25 @@
 env_files = set()
 archive_settings = {}
 
-is_comment = re.compile(r'\s*#.*').match
+is_comment = re.compile(r'[ \t]*#.*').match
 # # nuclio: return
-is_return = re.compile(r'#\s*(nuclio|mlrun):\s*return').search
+is_return = re.compile(r'#[ \t]*(nuclio|mlrun):[ \t]*return').search
 # # nuclio: ignore
-has_ignore = re.compile(r'#\s*(nuclio|mlrun):\s*ignore').search
-has_start = re.compile(r'#\s*(nuclio|mlrun):\s*start-code').search
-has_end = re.compile(r'#\s*(nuclio|mlrun):\s*end-code').search
+has_ignore = re.compile(r'#[ \t]*(nuclio|mlrun):[ \t]*ignore').search
+has_start = re.compile(
+    r'#[ \t]*(nuclio|mlrun):[ \t]*start-code[ \t]*(?P<name>([\S]*))?'
+).search
+has_end = re.compile(
+    r'#[ \t]*(nuclio|mlrun):[ \t]*end-code[ \t]*(?P<name>([\S]*))?'
+).search
 handler_decl = 'def {}(context, event):'
 indent_prefix = '    '
 line_magic = '%nuclio'
 cell_magic = '%' + line_magic
+closed = 'closed'
+started = 'started'
+code_cells = 'code_cells'
+nameless_annotation = ''
```

Comment:
These things don't need to be globals (the other are also hideous but I'm trying to not make this a refactor PR).
you can define them in `from_notebook_node` - they are used just there, or just use the raw strings even - because this is used in a very small scope.
in any case it's not worth adding globals here for every string constant var we want in some function